app/models/models.scala

```diff
@@ -12,7 +14,7 @@ case class Talk(
   url: String,
   authors: Seq[Author],
   location: String,
-  date: DateTime,
+  date: String,
```

Comment:
The `flatMap(_.toOption)` in https://github.com/guardian/digital-department-website/pull/9/files#r77160514 means any parsing errors will just mean any errors trying to read from Scanamo will just disappear into the void. If you keep the Xor structure you will get something more informative about what the problem was.